tensorboard/dataclass_compat_test.py

```diff
@@ -212,6 +215,49 @@ def test_graph_def(self):
 
         self.assertProtoEquals(graph_def, new_graph_def)
 
+    def test_graph_def_experimental_filter_graph(self):
+        # Create a `GraphDef` and write it to disk as an event.
+        logdir = self.get_temp_dir()
```

Comment:
Question about testing strategy: is it really necessary to create a logdir and a writer? If your intention here is to simply test `self._migrate_event()` (and ultimately `dataclass_compat.migrate_event()`), you can just construct a `Event`, pass it to that method and check the result, right?

Reply:
Oh good point.  I had pasted that from the test above but here the logdir part is superfluous.  (There too, maybe, but that's out of scope).  Thanks!